### DISCUSSION TITLE
Spyhunter & SubtitleEdit

### DISCUSSION THREAD

**Discussion by zorrax2b** [SubtitleEdit/subtitleedit#4919] [General]
Hello
just to inform you that the last version of SubtitleEdit was detected by Spyhunter with Trojan Msil.Krypt
C:\Program Files\Subtitle Edit\SubtitleEdit.exe
MD5: de58207cc0c61d3a7579f9810fc5a8db

**Comment by niksedk** [comment]
Please report false positive to your anti virus provider, thx !

Do try https://www.virustotal.com to check multiple providers: https://www.virustotal.com/gui/file/06011aa19ae77aa82043fd92579d05d9cbe48069c12aff9e197ec36ba0eeb418/detection
